### ISSUE TITLE
🔨 prepare a demostack

### ISSUE THREAD

**Issue by davidB** [cdviz-dev/cdviz#6]
The demo/test stack could deploy on side of the application:

- optional installation of cdviz (in case already installed by skaffold for local dev, or ...)
- optional installation of some providers (each in its own namespace, via its own chart, ...)
  - testkube: [Emitting Standard CDEvents | Testkube Documentation](https://docs.testkube.io/articles/cd-events/)
  - harbor
  - minio
  - ???
- installation via just (for dev), and wrapp a call to helmfile or [helmwave](https://docs.helmwave.app/)
- just commands to trigger activity (load data, build stuff, fake events...)

**Comment by rjtch** [comment]
can i work on this task?

**Comment by davidB** [comment]
> can i work on this task?

No, I would like to make the initial setup. I also want to give a try to helmwave. And this issue will probably be split into smaller issues, task.
